lib/ui/hooks.dart

```diff
@@ -162,6 +162,20 @@ void _drawFrame() {
   _invoke(window.onDrawFrame, window._onDrawFrameZone);
 }
 
+@pragma('vm:entry-point')
+// ignore: unused_element
+void _runMainZoned(Function startMainIsolateFunction, Function userMainFunction) {
+  startMainIsolateFunction((){
+    runZoned<Future<void>>(() async {
```

Comment:
Does this closure need to be async?
runZoned<void>(() { ... ?

Reply:
I dont think so.

Reply:
Patched.